### PR TITLE
Translate recent news posts (ru)

### PR DESCRIPTION
**Pull request by shprotru**
added some news entries Russian translations

### CODE REVIEW

File: ru/news/_posts/2019-04-17-ruby-2-6-3-released.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: news_post
+title: "Вышел Ruby 2.6.3"
+author: "shprotru"
```
Comment (stomar) [inline ru/news/_posts/2019-04-17-ruby-2-6-3-released.md]: author must not be changed

File: ru/news/_posts/2019-08-28-multiple-jquery-vulnerabilities-in-rdoc.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+layout: news_post
+title: "Множественные уязвимости jQuery в RDoc"
+author: "aycabta"
+translator: "shprotru"
+date: 2019-08-28 09:00:00 +0000
+tags: безопасность
```
Comment (stomar) [inline ru/news/_posts/2019-08-28-multiple-jquery-vulnerabilities-in-rdoc.md]: `security` tag must be kept unchanged

File: ru/news/_posts/2019-10-01-http-response-splitting-in-webrick-cve-2019-16254.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+layout: news_post
+title: "CVE-2019-16254: Разделение HTTP-ответа в WEBrick (Дополняющее исправление)"
+author: "mame"
+translator: "shprotru"
+date: 2019-10-01 11:00:00 +0000
+tags: безопасность
```
Comment (stomar) [inline ru/news/_posts/2019-10-01-http-response-splitting-in-webrick-cve-2019-16254.md]: wrong tag

File: ru/news/_posts/2019-10-01-nul-injection-file-fnmatch-cve-2019-15845.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+layout: news_post
+title: "CVE-2019-15845: Уязвимость NUL инъекции в File.fnmatch и File.fnmatch?"
+author: "mame"
+translator: "shprotru"
+date: 2019-10-01 11:00:00 +0000
+tags: безопасность
```
Comment (stomar) [inline ru/news/_posts/2019-10-01-nul-injection-file-fnmatch-cve-2019-15845.md]: wrong tag

File: ru/news/_posts/2019-04-17-ruby-2-6-3-released.md (hunk 1 above)
Comment (shprotru) [reply ru/news/_posts/2019-04-17-ruby-2-6-3-released.md]: fixed

File: ru/news/_posts/2019-08-28-multiple-jquery-vulnerabilities-in-rdoc.md (hunk 2 above)
Comment (shprotru) [reply ru/news/_posts/2019-08-28-multiple-jquery-vulnerabilities-in-rdoc.md]: fixed

File: ru/news/_posts/2019-10-01-http-response-splitting-in-webrick-cve-2019-16254.md (hunk 3 above)
Comment (shprotru) [reply ru/news/_posts/2019-10-01-http-response-splitting-in-webrick-cve-2019-16254.md]: fixed

File: ru/news/_posts/2019-10-01-nul-injection-file-fnmatch-cve-2019-15845.md (hunk 4 above)
Comment (shprotru) [reply ru/news/_posts/2019-10-01-nul-injection-file-fnmatch-cve-2019-15845.md]: fixed

File: ru/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+`irb`, включаемая интерактивная оболочка (REPL; Read-Eval-Print-Loop), теперь поддерживает многострочное редактирование. Оно работает на `reline`, совместимой с `readline` но реализованной на Ruby.
+Обеспечена интеграция rdoc. Теперь в `irb` можно отобразить ссылку на указанный класс, модуль или метод.  [#14683](https://bugs.ruby-lang.org/issues/14683), [#14787](https://bugs.ruby-lang.org/issues/14787), [#14918](https://bugs.ruby-lang.org/issues/14918)
+Кроме того, строки исходников в `binding.irb` и результаты инспектирования для базовых классов теперь подсвечиваются.
+
+<video autoplay="autoplay" controls="controls" muted="muted" width="576" height="259">
+  <source src="https://cache.ruby-lang.org/pub/media/irb_improved_with_key_take2.mp4" type="video/mp4">
+</video>
+
+## Другие Примечательные Нововведения
```
Comment (lex111) [inline ru/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md]: Здесь и далее слова в заголовках с маленькой буквы, кроме первого слова.

**Comment by stomar** [comment]
For merging: should be merged with "Squash and merge" and PR title as commit message.

**Comment by lex111** [comment]
@stomar could you please see why tests fail in the `master` branch?

**Pull request merged**